### DISCUSSION TITLE
option to add rules such as [biome] must be touching [biome]

### DISCUSSION THREAD

**Discussion by maxinefinnfoxen** [Zodsmar/SeedSearcherStandaloneTool#98] [Ideas]
an option to add rules the biomes must follow would be very handy for finding islands made of certain biomes and much more. to find a jungle with a mesa in the middle of the island you would select rules that make this true:
1. mesa must not be touching any biomes other than jungle 
3. jungle must not be touching any biomes other than mesa and ocean
for more success this could be searched for anywhere within 60,000 by 60,000 from spawn, not just at spawn.

**Comment by Zodsmar** [comment]
Specific Rules is something I want to add in the future
